addons/background/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Looks like we have a SSOT for licenses: https://github.com/storybooks/storybook/blob/master/LICENSE